src/framework/tooltip/_tooltip.scss

```diff
@@ -0,0 +1,50 @@
+
+$fieldMessage_arrowSide: 6px;
+$fieldMessage_bgColor: #fafdfe;
+$fieldMessage_borderColor: #dbe2e3;
```

Comment:
Background-color and border-color are taken from the mock ups from Nithin from https://projects.invisionapp.com/d/main#/console/11263450/239311720/inspect